06-raster-vector.Rmd

```diff
@@ -479,6 +479,10 @@ grain_poly = as.polygons(grain) |>
   st_as_sf()
 ```
 
+The aggregated polygons of the `grain` dataset have rectilinear boundaries which arise from being defined by the boundaries of pixels.
+The **smoothr** package described in Chapter \@ref(geometry-operations) can be used to smooth the edges of the polygons.
+The smoothed polygons will not have the same exact spatial footprint as the original pixels, so caution should be used for using the smoothed polygons for further analysis.
```

Comment:
What does the "spatial footprint" mean?

Reply:
Changed this sentence to be something more like "coverage" and made it more explicit, maybe this is clearer?